packages/component-library/package.json

```diff
@@ -131,6 +131,7 @@
     "stylelint-config-standard-scss": "^13.1.0",
     "stylelint-define-config": "^1.8.0",
     "ts-morph": "^20.0.0",
+    "type-fest": "^4.41.0",
```

Comment:
We use this dependency in our test. 

For some reason it got removed, I re-added it.